Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -829,6 +829,7 @@ class BrowserViewController: UIViewController {
     }
 
     func openURLInNewTab(url: NSURL) {
+        urlBar.leaveOverlayMode()
```

Comment:
I had to add this to get back from the home panel to the browser after opening a link in a new tab.